website/modules/home-simplified/public/css/home-simplified.css

```
@@ -0,0 +1,558 @@
+.ca-foot a:hover {
+  color: #141414;
+}
+
+.ca-foot .l,
+.ca-foot .r {
+  display: flex;
+  gap: 28px;
+  align-items: center;
+}
+
+@keyframes fadeUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)